### DISCUSSION TITLE
How to enable additional features during compilation?

### DISCUSSION THREAD

**Discussion by mskala1** [dbeaver/dbeaver#13809] [DBeaver Development]
Hi, How can I enable some additional features/plugins (like Git or Office) during compilation? Unfortunately, I'm not Java developer so please share some examples.
Regards
Mateusz

**Comment by LonwoLonwo** [comment]
Please tell me more about how you compile the project?
Plugins mentioned by you are already in the plugins list https://github.com/dbeaver/dbeaver/tree/devel/plugins
